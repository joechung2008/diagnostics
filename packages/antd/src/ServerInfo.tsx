import { Table } from "antd";

const ServerInfo: React.FC<ServerInfoProps> = ({
  deploymentId,
  extensionSync,
  hostname,
  nodeVersions,
  serverId,
  uptime,
}) => {
  const items = [
    {
      name: "Hostname",
      value: hostname,
    },
    {
      name: "Uptime",
      value: uptime,
    },
    {
      name: "Server ID",
      value: serverId,
    },
    {
      name: "Deployment ID",
      value: deploymentId,
    },
    {
      name: "Node Versions",
      value: nodeVersions,
    },
    {
      name: "Extension Sync | Total Sync All Count",
      value: extensionSync.totalSyncAllCount,
    },
  ];

  const columns = [
    {
      title: "Name",
      dataIndex: "name",
      key: "name",
    },
    {
      title: "Value",
      dataIndex: "value",
      key: "value",
    },
  ];

  return (
    <Table
      aria-label="Server Info"
      columns={columns}
      dataSource={items.map((item) => ({ ...item, key: item.name }))}
      pagination={false}
    />
  );
};

export default ServerInfo;
