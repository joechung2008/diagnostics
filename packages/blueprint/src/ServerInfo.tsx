import { HTMLTable } from "@blueprintjs/core";

const ServerInfo: React.FC<ServerInfoProps> = ({
  deploymentId,
  extensionSync,
  hostname,
  nodeVersions,
  serverId,
  uptime,
}) => {
  const items = [
    {
      name: "Hostname",
      value: hostname,
    },
    {
      name: "Uptime",
      value: uptime,
    },
    {
      name: "Server ID",
      value: serverId,
    },
    {
      name: "Deployment ID",
      value: deploymentId,
    },
    {
      name: "Node Versions",
      value: nodeVersions,
    },
    {
      name: "Extension Sync | Total Sync All Count",
      value: extensionSync.totalSyncAllCount,
    },
  ];

  return (
    <HTMLTable aria-label="Server Info">
      <thead>
        <tr>
          <th>Name</th>
          <th>Value</th>
        </tr>
      </thead>
      <tbody>
        {items.map((item) => (
          <tr key={item.name}>
            <td>{item.name}</td>
            <td>{item.value}</td>
          </tr>
        ))}
      </tbody>
    </HTMLTable>
  );
};

export default ServerInfo;
