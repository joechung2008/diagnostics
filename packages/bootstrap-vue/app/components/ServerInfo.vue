<template>
  <div>
    <BTable :items="rows" :fields="fields" small />
  </div>
</template>

<script setup lang="ts">
import type { ServerInfoProps } from "@/types/ServerInfoProps";
import { when } from "@/utils/extensions";

const props = defineProps<ServerInfoProps>();

const fields = [
  { key: "name", label: "Name" },
  { key: "value", label: "Value" }
];

const rows = computed(() => [
  { name: "Hostname", value: props.hostname },
  ...when(props.uptime !== undefined, {
    name: "Uptime",
    value: String(props.uptime)
  }),
  { name: "Server ID", value: props.serverId },
  { name: "Deployment ID", value: props.deploymentId },
  ...when(props.nodeVersions !== undefined, {
    name: "Node Versions",
    value: props.nodeVersions
  }),
  {
    name: "Extension Sync | Total Sync All Count",
    value: String(props.extensionSync.totalSyncAllCount)
  }
]);
</script>
