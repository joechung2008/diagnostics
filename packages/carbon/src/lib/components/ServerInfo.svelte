<script lang="ts">
	import { DataTable } from 'carbon-components-svelte';

	const { deploymentId, extensionSync, hostname, nodeVersions, serverId, uptime }: ServerInfoProps =
		$props();

	const headers = [
		{ key: 'name', value: 'Name', empty: false },
		{ key: 'value', value: 'Value', empty: false }
	];

	const rows = $derived([
		{ id: 'hostname', name: 'Hostname', value: hostname },
		{ id: 'uptime', name: 'Uptime', value: uptime },
		{ id: 'server-id', name: 'Server ID', value: serverId },
		{ id: 'deployment-id', name: 'Deployment ID', value: deploymentId },
		{ id: 'node-versions', name: 'Node Versions', value: nodeVersions },
		{
			id: 'extension-sync',
			name: 'Extension Sync | Total Sync All Count',
			value: extensionSync.totalSyncAllCount
		}
	]);
</script>

<DataTable aria-label="Server Information" {headers} {rows} />
