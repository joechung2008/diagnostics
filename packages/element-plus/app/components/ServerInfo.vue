<template>
    <el-table :data="rows" style="width: 100%" aria-label="Server Info">
        <el-table-column prop="name" label="Name" show-overflow-tooltip />
        <el-table-column prop="value" label="Value" show-overflow-tooltip />
    </el-table>
</template>

<script setup lang="ts">
import type { ServerInfoProps } from "@/types"

const props = defineProps<ServerInfoProps>()

const rows = computed(() => [
    { name: "Hostname", value: props.hostname },
    { name: "Uptime", value: props.uptime },
    { name: "Server ID", value: props.serverId },
    { name: "Deployment ID", value: props.deploymentId },
    { name: "Node Versions", value: props.nodeVersions },
    {
        name: "Extension Sync | Total Sync All Count",
        value: props.extensionSync.totalSyncAllCount
    }
])
</script>
