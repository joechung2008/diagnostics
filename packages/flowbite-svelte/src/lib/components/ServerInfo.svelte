<script lang="ts">
  import {
    Table,
    TableBody,
    TableBodyCell,
    TableBodyRow,
    TableHead,
    TableHeadCell
  } from "flowbite-svelte";

  let { deploymentId, extensionSync, hostname, nodeVersions, serverId, uptime }: ServerInfoProps =
    $props();

  const items = $derived.by(() => [
    { name: "Hostname", value: hostname },
    { name: "Uptime", value: uptime },
    { name: "Server ID", value: serverId },
    { name: "Deployment ID", value: deploymentId },
    { name: "Node Versions", value: nodeVersions },
    {
      name: "Extension Sync | Total Sync All Count",
      value: extensionSync.totalSyncAllCount
    }
  ]);
</script>

<Table aria-label="Server Info">
  <TableHead>
    <TableHeadCell>Name</TableHeadCell>
    <TableHeadCell>Value</TableHeadCell>
  </TableHead>
  <TableBody>
    {#each items as item (item.name)}
      <TableBodyRow>
        <TableBodyCell>{item.name}</TableBodyCell>
        <TableBodyCell>{item.value}</TableBodyCell>
      </TableBodyRow>
    {/each}
  </TableBody>
</Table>
