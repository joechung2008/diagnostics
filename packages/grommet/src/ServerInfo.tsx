import { Table, TableBody, TableCell, TableHeader, TableRow } from "grommet";

const ServerInfo: React.FC<ServerInfoProps> = ({
  deploymentId,
  extensionSync,
  hostname,
  nodeVersions,
  serverId,
  uptime,
}) => {
  const items = [
    {
      name: "Hostname",
      value: hostname,
    },
    {
      name: "Uptime",
      value: uptime,
    },
    {
      name: "Server ID",
      value: serverId,
    },
    {
      name: "Deployment ID",
      value: deploymentId,
    },
    {
      name: "Node Versions",
      value: nodeVersions,
    },
    {
      name: "Extension Sync | Total Sync All Count",
      value: extensionSync.totalSyncAllCount,
    },
  ];

  return (
    <Table aria-label="Server Info">
      <TableHeader>
        <TableRow>
          <TableCell scope="col">Name</TableCell>
          <TableCell scope="col">Value</TableCell>
        </TableRow>
      </TableHeader>
      <TableBody>
        {items.map((item) => (
          <TableRow key={item.name}>
            <TableCell>{item.name}</TableCell>
            <TableCell>{item.value}</TableCell>
          </TableRow>
        ))}
      </TableBody>
    </Table>
  );
};

export default ServerInfo;
