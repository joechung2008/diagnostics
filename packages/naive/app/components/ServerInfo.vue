<template>
  <n-data-table :columns="columns" :data="rows" size="small" />
</template>

<script setup lang="ts">
import type { ServerInfoProps } from "@/types/ServerInfoProps"
import { when } from "~/utils/extensions"

const props = defineProps<ServerInfoProps>()

const columns = [
  { title: "Name", key: "name" },
  { title: "Value", key: "value" }
]

const rows = computed(() => [
  { name: "Hostname", value: props.hostname },
  ...when(props.uptime !== undefined, {
    name: "Uptime",
    value: String(props.uptime)
  }),
  { name: "Server ID", value: props.serverId },
  { name: "Deployment ID", value: props.deploymentId },
  ...when(props.nodeVersions !== undefined, {
    name: "Node Versions",
    value: props.nodeVersions
  }),
  {
    name: "Extension Sync | Total Sync All Count",
    value: String(props.extensionSync.totalSyncAllCount)
  }
])
</script>
