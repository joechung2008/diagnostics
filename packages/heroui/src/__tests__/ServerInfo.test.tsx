import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import ServerInfo from "../ServerInfo";

describe("ServerInfo", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(
      <ServerInfo
        deploymentId="test-deployment"
        extensionSync={{ totalSyncAllCount: 0 }}
        hostname="localhost"
        nodeVersions="v18.0.0"
        serverId="test-server"
        uptime={12345}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
