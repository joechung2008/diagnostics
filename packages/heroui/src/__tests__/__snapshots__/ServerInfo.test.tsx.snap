// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ServerInfo > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col relative gap-4 w-full"
  >
    <div
      class="p-4 z-0 flex flex-col relative justify-between gap-4 bg-content1 overflow-auto shadow-small rounded-large w-full"
    >
      <table
        aria-describedby=""
        aria-label="Server Information"
        class="min-w-full h-auto table-auto w-full"
        data-collection="react-aria-_r_1_"
        id="react-aria-_r_0_"
        role="table"
        tabindex="0"
      >
        <thead
          class="[&>tr]:first:rounded-lg"
          role="rowgroup"
        >
          <tr
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            role="row"
          >
            <th
              aria-colindex="1"
              class="group/th px-3 h-10 align-middle bg-default-100 whitespace-nowrap text-foreground-500 text-tiny font-semibold first:rounded-s-lg last:rounded-e-lg data-[sortable=true]:cursor-pointer data-[hover=true]:text-foreground-400 outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-start"
              data-collection="react-aria-_r_1_"
              data-key="$.0"
              data-react-aria-pressable="true"
              id="server-name-header"
              role="columnheader"
              scope="col"
              tabindex="-1"
            >
              Name
            </th>
            <th
              aria-colindex="2"
              class="group/th px-3 h-10 align-middle bg-default-100 whitespace-nowrap text-foreground-500 text-tiny font-semibold first:rounded-s-lg last:rounded-e-lg data-[sortable=true]:cursor-pointer data-[hover=true]:text-foreground-400 outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 text-start"
              data-collection="react-aria-_r_1_"
              data-key="$.1"
              data-react-aria-pressable="true"
              id="server-value-header"
              role="columnheader"
              scope="col"
              tabindex="-1"
            >
              Value
            </th>
          </tr>
          <tr
            aria-hidden="true"
            class="w-px h-px block"
            style="margin-left: 0.25rem; margin-top: 0.25rem;"
            tabindex="-1"
          />
        </thead>
        <tbody
          class="after:block"
          role="rowgroup"
        >
          <tr
            aria-labelledby="react-aria-_r_0_-Hostname-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-first="true"
            data-key="Hostname"
            id="react-aria-_r_5_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Hostname.0"
              headers="server-name-header"
              id="react-aria-_r_0_-Hostname-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Hostname
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Hostname.1"
              headers="server-value-header"
              id="react-aria-_r_7_"
              role="cell"
              tabindex="-1"
            >
              <span>
                localhost
              </span>
            </td>
          </tr>
          <tr
            aria-labelledby="react-aria-_r_0_-Uptime-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-key="Uptime"
            data-middle="true"
            data-odd="true"
            id="react-aria-_r_8_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Uptime.0"
              headers="server-name-header"
              id="react-aria-_r_0_-Uptime-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Uptime
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Uptime.1"
              headers="server-value-header"
              id="react-aria-_r_a_"
              role="cell"
              tabindex="-1"
            >
              <span>
                12345
              </span>
            </td>
          </tr>
          <tr
            aria-labelledby="react-aria-_r_0_-ServerID-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-key="Server ID"
            data-middle="true"
            id="react-aria-_r_b_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Server ID.0"
              headers="server-name-header"
              id="react-aria-_r_0_-ServerID-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Server ID
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Server ID.1"
              headers="server-value-header"
              id="react-aria-_r_d_"
              role="cell"
              tabindex="-1"
            >
              <span>
                test-server
              </span>
            </td>
          </tr>
          <tr
            aria-labelledby="react-aria-_r_0_-DeploymentID-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-key="Deployment ID"
            data-middle="true"
            data-odd="true"
            id="react-aria-_r_e_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Deployment ID.0"
              headers="server-name-header"
              id="react-aria-_r_0_-DeploymentID-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Deployment ID
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Deployment ID.1"
              headers="server-value-header"
              id="react-aria-_r_g_"
              role="cell"
              tabindex="-1"
            >
              <span>
                test-deployment
              </span>
            </td>
          </tr>
          <tr
            aria-labelledby="react-aria-_r_0_-NodeVersions-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-key="Node Versions"
            data-middle="true"
            id="react-aria-_r_h_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Node Versions.0"
              headers="server-name-header"
              id="react-aria-_r_0_-NodeVersions-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Node Versions
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Node Versions.1"
              headers="server-value-header"
              id="react-aria-_r_j_"
              role="cell"
              tabindex="-1"
            >
              <span>
                v18.0.0
              </span>
            </td>
          </tr>
          <tr
            aria-labelledby="react-aria-_r_0_-ExtensionSync|TotalSyncAllCount-$.0"
            class="group/tr outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2"
            data-collection="react-aria-_r_1_"
            data-key="Extension Sync | Total Sync All Count"
            data-last="true"
            data-odd="true"
            id="react-aria-_r_k_"
            role="row"
            tabindex="-1"
          >
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Extension Sync | Total Sync All Count.0"
              headers="server-name-header"
              id="react-aria-_r_0_-ExtensionSync|TotalSyncAllCount-$.0"
              role="cell"
              tabindex="-1"
            >
              <span>
                Extension Sync | Total Sync All Count
              </span>
            </td>
            <td
              class="py-2 px-3 relative align-middle whitespace-normal text-small font-normal [&>*]:z-1 [&>*]:relative outline-solid outline-transparent data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 before:pointer-events-none before:content-[''] before:absolute before:z-0 before:inset-0 before:opacity-0 data-[selected=true]:before:opacity-100 group-data-[disabled=true]/tr:text-foreground-300 group-data-[disabled=true]/tr:cursor-not-allowed before:bg-default/60 data-[selected=true]:text-default-foreground first:before:rounded-s-lg last:before:rounded-e-lg text-start"
              data-collection="react-aria-_r_1_"
              data-key="Extension Sync | Total Sync All Count.1"
              headers="server-value-header"
              id="react-aria-_r_m_"
              role="cell"
              tabindex="-1"
            >
              <span>
                0
              </span>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</DocumentFragment>
`;
